.github/workflows/linux-recreate-runner.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   recreate:
+    if:  startsWith(github.repository, 'Homebrew/')
```

Comment:
I think we prefer `if: github.repository == 'Homebrew/homebrew-core`, as we use that in other workflows.